test/lib/sdkdriver/src/dmp.ts

```diff
@@ -0,0 +1,132 @@
+import { EventEmitter } from 'events';
+import Transport from './transport';
+
+/**
+ * DMP Request.
+ * @class
+ * @classdesc A {@link DMPRequest} contains the data associated with
+ *   the incoming request message along with a method for sending a
+ *   response.
+ */
+class DMPRequest {
```

Comment:
we need to export this, right?

Reply:
@markandrus I don't think so, since the constructor is called internally when we are emitting a "request" event on DMP.

Reply:
How does a consumer of `DMP` use the `DMPRequest`, though? They want to be able to write something like this, right?

```ts
import DMP, { DMPRequest } from './dmp'

const dmp = new DMP(/* ... */)
dmp.on('request', (request: DMPRequest) => {
  // ...
})
```